src/nvc.c

```diff
@@ -120,13 +138,16 @@ static int analyse(int argc, char **argv)
       { "psl",             no_argument,       0, 'P' },
       { "relax",           required_argument, 0, 'X' },
       { "relaxed",         no_argument,       0, 'R' },
+      { "define",          required_argument, 0, 'd' },
       { 0, 0, 0, 0 }
    };
 
    const int next_cmd = scan_cmd(2, argc, argv);
    int c, index = 0;
    const char *spec = ":";
```

Comment:
```suggestion
   const char *spec = ":D:";
```

Let's also support `DKEY=VALUE`.

Reply:
See the comment above. Should I change the `dump-llvm` to something else than `D` ?

Reply:
Actually that option can be deleted: we don't generate any LLVM at analysis time any more.